packages/java/src/node/java-contribution.ts

```diff
@@ -48,7 +48,7 @@ export class JavaContribution extends BaseLanguageServerContribution {
 
         if (DEBUG_MODE) {
             args.push(
-                '-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=1044',
+                '-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=1044',
```

Comment:
@gorkem, do we really want to suspend the Jvm? I guess it's useful to debug the LS handshake. Just asking.

Reply:
Good catch. It should be fixed now.